### ISSUE TITLE
iOS 9 support?

### ISSUE THREAD

**Issue by Spacelapp** [nxtbgthng/OAuth2Client#194]
NSURLConnection is deprecated

**Comment by Spacelapp** [comment]
Hello?

**Comment by toto** [comment]
@Spacelapp Yes it is deprecated and should be removed. Unfortunately getting rid of NSURLConnection would require a big change to the library. This is something I can't tackle at the moment. In general the library needs major updates (e.g. Swift support annotations).

**Comment by Spacelapp** [comment]
okay :(

**Comment by Idefix60** [comment]
Starting June 1, 2016 all apps submitted to the App Store must support IPv6-only networking.
https://developer.apple.com/news/?id=05042016a
OAuth2Client is not compatible with IPv6-only => no app update with the current OAuth2Client version from June 1.

**Comment by Muesly** [comment]
Is that true OAuth2Client is not compatible?  Seeing this makes me wonder if your only concern was NSURLConnection @Idefix60?: http://stackoverflow.com/questions/37067619/do-i-need-to-replace-nsurlconnection-in-order-to-achieve-mandatory-support-for-i
